_posts/2020-11-30-developer-survey-2020.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Solita Developer Survey 2020 - what technologies do we use and like?
+author: juholei
+excerpt: >
+  todo
+tags:
+ - Software Development
+
+---
+
+Developer survey has now become a tradition here at Solita. Every two years we survey our developer-minded people regarding what technologies and practices we use in our every-day work. The first survey was conducted in 2016 and you can find a blog post about those results in [our previous blog post](https://dev.solita.fi/java/2016/05/13/Developers-love-spaces.html) and then in 2018 [(blog post in Finnish)](https://www.solita.fi/blogit/development-teknologiakartoitus/). This year we once again took a loot at the survey and came up with a total of 27 questions that would give us interesting insight into what we use, what we like and what could be big in the future. Comparing these results to the previous ones can also give us some information on trends - what technologies are increasing in use and what are in decline?
```

Comment:
It would be interesting to know how many people participated in the survey and maybe some other background info as well like what kind of developing they do etc.